packages/bpk-component-calendar-scrollable/package-lock.json

```diff
@@ -0,0 +1,291 @@
+{
+	"name": "bpk-component-calendar-scrollable",
+	"version": "0.0.0",
+	"lockfileVersion": 1,
+	"requires": true,
+	"dependencies": {
+		"asap": {
+			"version": "2.0.6",
+			"resolved": "https://registry.npmjs.org/asap/-/asap-2.0.6.tgz",
+			"integrity": "sha1-5QNHYR1+aQlDIIu9r+vLwvuGbUY="
+		},
+		"bpk-component-calendar": {
```

Comment:
Backpack dependencies (`bpk-*`) shouldn't appear in the `package-lock.json`. We rely on [lerna](https://github.com/lerna/lerna) to symlink these for development.

To fix:
1. `rm packages/bpk-component-calendar-scrollable/package-lock.json`
2. `rm -rf packages/bpk-component-calendar-scrollable/node_modules`
3. `npm run bootstrap`

That should regenerate